crates/control/src/rule_obstacle_composer.rs

```diff
@@ -122,11 +122,11 @@ pub fn create_penalty_box(field_dimensions: &FieldDimensions, kicking_team: Team
     };
     let half_field_length = field_dimensions.length / 2.0;
     let half_penalty_area_length = field_dimensions.penalty_area_length / 2.0;
-    let center_x = side_factor * (half_field_length - half_penalty_area_length);
+    let center_x = side_factor * (half_field_length - half_penalty_area_length + 0.1);
     RuleObstacle::Rectangle(Rectangle::new_with_center_and_size(
         point![center_x, 0.0],
         vector![
-            field_dimensions.penalty_area_length,
+            field_dimensions.penalty_area_length + 0.2,
```

Comment:
Please convert these to modifiable parameters ;)

Reply:
Did so, and I grouped the rule obstacle relevant parameters into one part of the ```default.json```. Do you think I should make a struct to group these further?

Reply:
Nice, is fine for me like this 👍🏻